tests/store/test_footprints.py

```diff
@@ -235,23 +299,29 @@ def test_read_footprint_standard():
     assert footprint_coords == ["height", "lat", "lev", "lon", "time"]
 
     assert "fp" in footprint_data.data_vars
+    assert "mean_age_particles_n" in footprint_data.data_vars
+    assert "mean_age_particles_e" in footprint_data.data_vars
+    assert "mean_age_particles_s" in footprint_data.data_vars
+    assert "mean_age_particles_w" in footprint_data.data_vars
 
     expected_attrs = {
         "author": "OpenGHG Cloud",
         "data_type": "footprints",
-        "site": "tac",
-        "height": "100m",
+        "site": "wao",
+        "height": "20m",
         "model": "NAME",
-        "domain": "europe",
-        "start_date": "2012-08-01 00:00:00+00:00",
-        "end_date": "2012-08-31 23:59:59+00:00",
-        "max_longitude": 39.38,
-        "min_longitude": -97.9,
-        "max_latitude": 79.057,
-        "min_latitude": 10.729,
+        "species": "rn",  # TODO: May want to see if we can keep this capitalised?
```

Comment:
I've had the same issue with some URLs taken from the ICOS metadata. At the moment `Datasource.add_metadata` calls `to_lowercase` which can break case sensitive things. I've just opened https://github.com/openghg/openghg/issues/333 to track this.